### PR TITLE
Started the Restructuring of the Development Project file

### PR DESCRIPTION
**Pull request by SeastEon**
List of new Files with the functions or classes moved in are: testing_operations
- StMTestImagePrep
- StMTestConnectionOperator
- DoImg
- StMTestGetFileFromDbFromUserId
- StMTestDeleteFileFromDbFromUserId
- StMTestSaveFileToDb
- outline_image

ui_operations
- Reset_Input_Images
- PlaceImageIn3D

### CODE REVIEW

**Comment by SeastEon** [comment]
This is just restructuring the project files.

File: ui_operations.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                    bpy.ops.import_image.to_plane(files=[{"name":filename, "name":filename}], directory=FileDirectory, relative=False)
+                    #we set the rotation and location of each plane
+                    bpy.data.objects[ImgName].select_set(True)
+                    match Itervalue :
+                        case 1: bpy.ops.transform.translate(value=(-0.01, 0 , 0), orient_type='GLOBAL', orient_matrix=((1, 0, 0), (0, 1, 0), (0, 0, 1)), orient_matrix_type='GLOBAL', constraint_axis=(False, True, False), mirror=False, use_proportional_edit=False, proportional_edit_falloff='SMOOTH', proportional_size=1, use_proportional_connected=False, use_proportional_projected=False, snap=False, snap_elements={'INCREMENT'}, use_snap_project=False, snap_target='CLOSEST', use_snap_self=True, use_snap_edit=True, use_snap_nonedit=True, use_snap_selectable=False, alt_navigation=True)
+                        case 2: bpy.context.object.rotation_euler[2] = 0
+                else:
+                    match Itervalue:
+                        case 0: MissingView = "FontView"
```
Comment (rfernandesdasilva) [inline ui_operations.py:94]: typo. w/e tbh, just using this so you guys can see that we can comment

**Comment by rfernandesdasilva** [comment]
Pretty good start. Now we have UI-operations that are going to deal with more complex functionality than just the UI, and init only has basic UI elements. We still have DatabaseLogin and other functions in there, but this is a great start. All testing buttons and functionality are moved to a new file, and we can easily remove it from the addon once it is ready to be published.

In the end, we expect to have a very clean __init__ that will probably only register buttons. It will be way easier to identify where we need to work, or where something is located. 

Another point I would like to talk about is the naming of functions in our files. Someone (I don't know who to be honest) started naming testing methods as StMTest[something]. I thought that was a pretty good idea and I followed up with that. @Klimentbehr @Velkrim-07 I believe this is a good thing and we should follow this from now on.

**Comment by rfernandesdasilva** [comment]
as far as I can see, you only left panels in init file. is there a reason for it, maybe a limitation of the code? does panels need to be defined in init? if yes, please let me know.

I think that init should remain very clean and only what is extremely necessary to remain there. 

**DatabaseLogin** should be part of ui-operations as it is a more complex function that does more than just defining UI-elements. @SeastEon

**Comment by rfernandesdasilva** [comment]
apart from that, excellent job in the pull request. a lot has been done, some minor stuff needs to be finished. I also would like to point out that your description of the PR was very professional and helpful, the commits as well.

**Comment by rfernandesdasilva** [comment]
Just reminding @SeastEon that you will probably have to move imports as well, as DbLoginRegister is dependent on both authentication.py and bcrypt_password, so remember to remove them from init and import them into ui-operations.

**Pull request closed without merging**